src/service/performance-impl.js

```diff
@@ -48,6 +48,49 @@ const VISIBILITY_CHANGE_EVENT = 'visibilitychange';
  */
 let TickEventDef;
 
+/**
+ * Enum for tick event labels
+ * @enum {string}
+ */
+export const Ticks = {
+  ACCESS_AUTHORIZATION: 'aaa',
+  ACCESS_AUTHORIZATION_VISIBLE: 'aaav',
+  ADS_LAYOUT_DELAY: 'adld',
+  BAD_FRAMES: 'bf',
+  BATTERY_DROP: 'bd',
+  CONTENT_LAYOUT_DELAY: 'cld',
+  CUMULATIVE_LAYOUT_SHIFT: 'cls',
+  CUMULATIVE_LAYOUT_SHIFT_2: 'cls-2',
+  DOCUMENT_READY: 'dr',
+  END_INSTALL_STYLES: 'e_is',
+  FIRST_CONTENTFUL_PAINT: 'fcp',
+  FIRST_CONTENTFUL_PAINT_VISIBLE: 'fcpv',
```

Comment:
Was the `fcp-v` to `fcpv` change intentional? Probably done to align with the other "visible" metrics, but just want to confirm that nothing needed `fcp-v`.

Reply:
@jridgewell Yes, it was intentional, I saw when I was adding the enum that there were some other ones suffixed `v` without the `-` for visible, so I went with it for the sake of being consistent.